src/config-hardware/mouse-touchpad.md

```diff
@@ -0,0 +1,65 @@
+
+![Mouse & Touchpad Settings](/images/config-mouse-touchpad/mouse-touchpad-settings.png)
+
+Mouse settings include Mouse Speed, Mouse Acceleration, and Natural Scrolling.
+
+Touchpad settings include enabling or disabling the touchpad entirely, Natural Scrolling, Touchpad Speed, Tap to Click, Two-finger scrolling, Edge Scrolling, and Disable while typing. Users can also set the Right or Left mouse button as the primary button.
+
+---
+
+## Testing Mouse Settings
+
+Click `Test Your Settings` in the upper right corner of the window. Use this screen to perform and observe normal navigational actions.
+
+![Mouse Testing Screen](/images/config-mouse-touchpad/test-mouse-settings.png)
+
+Scroll up and down on the mouse wheel or drive two fingers across the track pad to test scroll speed.
+
+<video autoplay loop>
+    <source src="/images/config-mouse-touchpad/test-scrolling.webm" />
+</video>
+
+Left, right, or middle-click the circle to test each assigned button and click registration speed. Test double-click speed by clicking the mouse button twice within the circle. The first click fills the outer ring, and the second click fills the circle.
+
+<video autoplay loop>
+    <source src="/images/config-mouse-touchpad/test-clicking.webm" />
+</video>
+
+## General
+
+The Primary Button performs primary actions in a graphical user interface (GUI). For right-handed users, this is usually assigned to the left mouse button.
+
+## Mouse
+
+| Option | Function |
+|--------|----------|
+| Mouse Speed | Adjust the speed of mouse cursor movement across the display when using a mouse. |
+| Mouse Acceleration | Enable or disable mouse speed increasing with sustained motion in any given direction. This allows less physical movement to reach further corners of the display. |
+| Natural Scrolling | Invert the direction content scrolls in response to the mouse wheel. When enabled, scrolling up on the mouse wheel pushes content toward the top of the window, moving the user's view to content lower on the page.
+
+## Touchpad
+
+| Option | Function |
+|--------|----------|
+| Touchpad | Enable or disable the touchpad. |
+| Touchpad Speed | Adjust the speed of mouse cursor movement across the display when using the touchpad. |
+| Disable while typing | Disable the touch pad when keyboard input is detected. |
+
+## Touchpad Click Options
+
+| Option | Function |
+|--------|----------|
+| Tap to Click | Enable single finger tap to left-click, two finger tap to right-click, and three finger tap to middle-click. |
+
+## Touchpad Scroll & Zoom Options
+
+| Option | Function |
+|--------|----------|
+| Natural Scrolling | When enabled, content is dragged in the direction of the input on the touch pad (similar to the behavior of the touch screen). |
+
+## Additional Mouse Settings
+
+Configure advanced mouse settings using the [GNOME Tweaks application](/customize-pop/gnome-tweaks-extensions/gnome-tweaks-keyboard-mouse.md).
```

Comment:
Do you think the advanced tweaks being in an earlier section than the basic options calls the organization of the book into question?

In any case, I would recommend placing a link back to this section at the top of the relevant page(s) of the Tweaks section so a user doesn't start going down rabbit holes they don't need to. Maybe something like "Note: Basic settings for \<xyz> can be applied in the [Settings application]."

Reply:
Yea good point. I suppose this section covers usage basics, so I'll open an issue in the repo as reminder to move this above Customize Pop.